osv/impact.py

```diff
@@ -43,6 +43,10 @@
 _DATASTORE_BATCH_SLEEP = 10
 
 
+class OrphanedCommitError(IndexError):
```

Comment:
Seems like we can handle this without needing to surface this exception right? Can we remove this?

Reply:
Oh yes. Removed in 219491c0